build/lib/run_target_docker.sh

```diff
@@ -21,7 +21,7 @@ PROJECT="$1"
 TARGET="$2"
 IMAGE_REPO="${3:-}"
 RELEASE_BRANCH="${4:-}"
-ARTIFACTS_BUCKET="${5:-}"
+ARTIFACTS_BUCKET="${5:-$ARTIFACTS_BUCKET}"
```

Comment:
This should be export'd and is not always passed in, for ex during the docker checksum build, but for kind this is needed to generate the config.